custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md

```diff
@@ -8,8 +8,13 @@ a [custom metrics setup] in your cluster.
 
 A simple sd-dummy-exporter container exports a metric of constant value 
 to Stackdriver in a loop. The metric name and value can be passed in via flags.
-Pod id is passed to the container via downward API (see [custom-metrics-sd deployment]
-for how it's done)
+Pod id, pod name and namespace are passed to the container via downward API (see
+[custom-metrics-sd deployment] for how it's done).
+
+Stackdriver dummy exporter can export metrics for **new Stackdriver resource model**,
+**legacy Stackdriver resource model** or both, specified by flags:
```

Comment:
Link to some description of resource models (can link to the adapter Readme)

Reply:
Will link a Stackdriver documentation once it's public.